src/structs/Api.ts

```diff
@@ -1,8 +1,9 @@
-import fetch, { Headers } from "node-fetch";
+import { fetch, Headers } from "undici";
```

Comment:
If we're going to use undici let's start using the `request()` method. We can also just merge in #74

Reply:
There's not really any good reason to use the `request()` method over the Fetch API, not only that but the `request()` method is also non-standard and differs from the Fetch API in multiple ways, Undici's `fetch()` method is also much more performant and standard compliant compared to Node-Fetch's.

The Fetch API is a standard which you can view [here](https://fetch.spec.whatwg.org/).

Reply:
If we're going to use the undici library we may as well use undici's request standard, I think it looks better anyways, let's use it.

Reply:
![image](https://user-images.githubusercontent.com/7953076/226073574-dd920c71-86f2-46f0-9fc9-c0ff82ecfc96.png)